### DISCUSSION TITLE
Search for Help b< Dahua DH-SD1A404XB-GNR PTZ Config (German/Englisch)

### DISCUSSION THREAD

**Discussion by 19Maverick79** [blakeblackshear/frigate#13655] [Config Support]
### Describe the problem you are having

Ich würde gerne die Dahua DH-SD1A404XB-GNR via PTZ und wenn möglich mit Autotracking in meinem Frigate integrieren. Als Anfänger weiß ich nur nicht wie ich das anstelle. Die Dokumentation im Bezug auf PTZ ist sehr wenig und im Internet gibt es keine brauchbaren Anleitungen z.B. bei YouTube. Kann mir jemand bitte helfen? Die Configuration habe ich anderswo kopiert. Die funktioniert, aber leider ohne PTZ und Tracking. Frigate läuft bei mir als Docker auf Unraid

I would like to integrate the Dahua DH-SD1A404XB-GNR into my frigate via PTZ and, if possible, with autotracking. As a beginner, I just don't know how to do it. There is very little documentation regarding PTZ and there are no useful instructions on the Internet, for example on YouTube. Can someone please help me? I copied the configuration elsewhere. It works, but unfortunately without PTZ and tracking. I run Frigate as Docker on Unraid

### Version

0.14

### Frigate config file

```yaml
mqtt:
  host: 192.168.212.5


database:
  path: /config/frigate.db

detectors:
  coral:
    type: edgetpu  
    device: usb

birdseye:
  enabled: True
  restream: True
  mode: objects

objects:
  track:
    - person

cameras:
  Hof:                                                            #Kameraname
    ffmpeg:
      inputs:
        - path: rtsp://admin:Passwort@192.168.212.105:554/cam/realmonitor?channel=1&subtype=0    #RTSP Stream High Res
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://admin:Passwort@192.168.212.105:554/cam/realmonitor?channel=1&subtype=1          #RTPS Stream Low Res
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      width: 1280
      height: 720
      fps: 5
    objects:
      track:
        - person
        
    snapshots:
      enabled: true
      timestamp: true
      bounding_box: true
      retain:
        default: 30
    record:
      enabled: true
      events:
        retain:
          default: 14
  Tuer:                                                            #Kameraname
    ffmpeg:
      inputs:
        - path: rtsp://admin:Passwort@192.168.212.110:554/cam/realmonitor?channel=1&subtype=0    #RTSP Stream High Res
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://admin:Passwort@192.168.212.110:554/cam/realmonitor?channel=1&subtype=1          #RTPS Stream Low Res
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      width: 1280
      height: 720
      fps: 5
    objects:
      track:
        - person
        
    snapshots:
      enabled: true
      timestamp: true
      bounding_box: true
      retain:
        default: 30
    record:
      enabled: true
      events:
        retain:
          default: 14
  Vorne:                                                            #Kameraname
    ffmpeg:
      inputs:
        - path: rtsp://admin:Passwort@192.168.212.107:554/cam/realmonitor?channel=1&subtype=0    #RTSP Stream High Res
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://admin:Passwort@192.168.212.107:554/cam/realmonitor?channel=1&subtype=1          #RTPS Stream Low Res
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      width: 1280
      height: 720
      fps: 5
    objects:
      track:
        - person
        
    snapshots:
      enabled: true
      timestamp: true
      bounding_box: true
      retain:
        default: 30
    record:
      enabled: true
      events:
        retain:
          default: 14
  Hinten:                                                            #Kameraname
    ffmpeg:
      inputs:
        - path: rtsp://admin:Passwort@192.168.212.106:554/cam/realmonitor?channel=1&subtype=0    #RTSP Stream High Res
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://admin:Passwort@192.168.212.106:554/cam/realmonitor?channel=1&subtype=1          #RTPS Stream Low Res
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      width: 1280
      height: 720
      fps: 5
    objects:
      track:
        - person
        
    snapshots:
      enabled: false
      timestamp: true
      bounding_box: true
      retain:
        default: 30
    record:
      enabled: false
      events:
        retain:
          default: 14

version: 0.14
```

### Relevant Frigate log output

```shell
2024-09-10 18:47:40.012863466  [INFO] Preparing Frigate...
2024-09-10 18:47:40.025205475  [INFO] Starting Frigate...
2024-09-10 18:47:41.240844459  [2024-09-10 18:47:41] frigate.app                    INFO    : Starting Frigate (0.14.1-f4f3cfa)
2024-09-10 18:47:41.240906759  [2024-09-10 18:47:41] frigate.util.config            INFO    : Checking if frigate config needs migration...
2024-09-10 18:47:41.275529203  [2024-09-10 18:47:41] frigate.util.config            INFO    : frigate config does not need migration...
2024-09-10 18:47:41.501022598  [2024-09-10 18:47:41] frigate.util.services          INFO    : Automatically detected vaapi hwaccel for video decoding
2024-09-10 18:47:41.540486776  [2024-09-10 18:47:41] peewee_migrate.logs            INFO    : Starting migrations
2024-09-10 18:47:41.540870757  [2024-09-10 18:47:41] peewee_migrate.logs            INFO    : There is nothing to migrate
2024-09-10 18:47:41.576982278  [2024-09-10 18:47:41] frigate.app                    INFO    : Recording process started: 429
2024-09-10 18:47:41.585131558  [2024-09-10 18:47:41] frigate.app                    INFO    : Recording process started: 438
2024-09-10 18:47:41.588568349  [2024-09-10 18:47:41] frigate.app                    INFO    : go2rtc process pid: 98
2024-09-10 18:47:41.620352786  [2024-09-10 18:47:41] detector.coral                 INFO    : Starting detection process: 458
2024-09-10 18:47:44.768380197  [2024-09-10 18:47:41] frigate.app                    INFO    : Output process started: 461
2024-09-10 18:47:44.768488957  [2024-09-10 18:47:41] frigate.detectors.plugins.edgetpu_tfl INFO    : Attempting to load TPU as usb
2024-09-10 18:47:44.768717115  [2024-09-10 18:47:41] frigate.app                    INFO    : Camera processor started for Hof: 478
2024-09-10 18:47:44.768788514  [2024-09-10 18:47:41] frigate.app                    INFO    : Camera processor started for Tuer: 481
2024-09-10 18:47:44.768857060  [2024-09-10 18:47:41] frigate.app                    INFO    : Camera processor started for Vorne: 488
2024-09-10 18:47:44.768926680  [2024-09-10 18:47:41] frigate.app                    INFO    : Camera processor started for Hinten: 495
2024-09-10 18:47:44.768992578  [2024-09-10 18:47:41] frigate.app                    INFO    : Capture process started for Hof: 508
2024-09-10 18:47:44.769064832  [2024-09-10 18:47:41] frigate.app                    INFO    : Capture process started for Tuer: 513
2024-09-10 18:47:44.769133527  [2024-09-10 18:47:41] frigate.app                    INFO    : Capture process started for Vorne: 517
2024-09-10 18:47:44.769202452  [2024-09-10 18:47:41] frigate.app                    INFO    : Capture process started for Hinten: 521
2024-09-10 18:47:44.772084959  [2024-09-10 18:47:44] frigate.detectors.plugins.edgetpu_tfl INFO    : TPU found
2024-09-10 18:48:06.945743563  [2024-09-10 18:48:06] frigate.util.services          ERROR   : Unable to poll intel GPU stats: Failed to initialize PMU! (Operation not permitted)
2024-09-10 18:49:08.531610  2024-09-10 18:48:06.945747091
```

### Relevant go2rtc log output

```shell
2024-09-10 18:47:40.016113862  [INFO] Preparing new go2rtc config...
2024-09-10 18:47:40.331562668  [INFO] Starting go2rtc...
2024-09-10 18:47:40.475130475  18:47:40.475 INF go2rtc platform=linux/amd64 revision=b2399f3 version=1.9.2
2024-09-10 18:47:40.475171302  18:47:40.475 INF config path=/dev/shm/go2rtc.yaml
2024-09-10 18:47:40.475453556  18:47:40.475 INF [api] listen addr=:1984
2024-09-10 18:47:40.475466895  18:47:40.475 INF [rtsp] listen addr=:8554
2024-09-10 18:47:40.475607401  18:47:40.475 INF [webrtc] listen addr=:8555/tcp
2024-09-10 18:47:50.012420340  [INFO] Starting go2rtc healthcheck service...
```

### Operating system

UNRAID

### Install method

Docker Compose

### Object Detector

Coral

**Comment by NickM-27** [comment]
the docs explain how to configure autotracking https://docs.frigate.video/configuration/autotracking#configuration

if you have questions regarding the docs then please be more specific on what is unclear

**Reply by 19Maverick79** [reply]
I used the lines of code provided there and added my information. But I get several errors when I click save and restart

`Config Error:

```
Traceback (most recent call last):
  File &#34;/opt/frigate/frigate/api/app.py&#34;, line 245, in config_save
    FrigateConfig.parse_raw(new_config)
  File &#34;/opt/frigate/frigate/config.py&#34;, line 1664, in parse_raw
    config = load_config_with_no_duplicates(raw_config)
  File &#34;/opt/frigate/frigate/util/builtin.py&#34;, line 116, in load_config_with_no_duplicates
    return yaml.load(raw_config, PreserveDuplicatesLoader)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/__init__.py&#34;, line 81, in load
    return loader.get_single_data()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/constructor.py&#34;, line 49, in get_single_data
    node = self.get_single_node()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 36, in get_single_node
    document = self.compose_document()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 55, in compose_document
    node = self.compose_node(None, None)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/parser.py&#34;, line 98, in check_event
    self.current_event = self.state()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/parser.py&#34;, line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/scanner.py&#34;, line 116, in check_token
    self.fetch_more_tokens()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/scanner.py&#34;, line 223, in fetch_more_tokens
    return self.fetch_value()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/scanner.py&#34;, line 577, in fetch_value
    raise ScannerError(None, None,
yaml.scanner.ScannerError: mapping values are not allowed here
  in &#34;&lt;unicode string&gt;&#34;, line 17, column 10:
        onvif:
             ^`

config.yaml

`mqtt:
  host: 192.168.212.5

detectors:
  coral:
    type: edgetpu  
    device: usb

birdseye:
  enabled: True
  restream: True
  mode: objects

cameras:
  ptzcamera:
    ...
    onvif:
      # Required: host of the camera being connected to.
      host: 192.168.212.106
      # Optional: ONVIF port for device (default: shown below).
      port: 8000
      # Optional: username for login.
      # NOTE: Some devices require admin to access ONVIF.
      user: admin
      # Optional: password for login.
      password: admin
      # Optional: PTZ camera object autotracking. Keeps a moving object in
      # the center of the frame by automatically moving the PTZ camera.
      autotracking:
        # Optional: enable/disable object autotracking. (default: shown below)
        enabled: False
        # Optional: calibrate the camera on startup (default: shown below)
        # A calibration will move the PTZ in increments and measure the time it takes to move.
        # The results are used to help estimate the position of tracked objects after a camera move.
        # Frigate will update your config file automatically after a calibration with
        # a "movement_weights" entry for the camera. You should then set calibrate_on_startup to False.
        calibrate_on_startup: False
        # Optional: the mode to use for zooming in/out on objects during autotracking. (default: shown below)
        # Available options are: disabled, absolute, and relative
        #   disabled - don't zoom in/out on autotracked objects, use pan/tilt only
        #   absolute - use absolute zooming (supported by most PTZ capable cameras)
        #   relative - use relative zooming (not supported on all PTZs, but makes concurrent pan/tilt/zoom movements)
        zooming: disabled
        # Optional: A value to change the behavior of zooming on autotracked objects. (default: shown below)
        # A lower value will keep more of the scene in view around a tracked object.
        # A higher value will zoom in more on a tracked object, but Frigate may lose tracking more quickly.
        # The value should be between 0.1 and 0.75
        zoom_factor: 0.3
        # Optional: list of objects to track from labelmap.txt (default: shown below)
        track:
          - person
        # Required: Begin automatically tracking an object when it enters any of the listed zones.
        required_zones:
          - zone_name
        # Required: Name of ONVIF preset in camera's firmware to return to when tracking is over. (default: shown below)
        return_preset: home
        # Optional: Seconds to delay before returning to preset. (default: shown below)
        timeout: 10
        # Optional: Values generated automatically by a camera calibration. Do not modify these manually. (default: shown below)
        movement_weights: []
```

**Reply by NickM-27** [reply]
the config is not formatted correctly, make sure tabs are not used. We really need a direct copy of the whole config to see what the issue might be

**Comment by 19Maverick79** [comment]
**Meine derzeitige Konfiguration / _my current configuration_**

`mqtt:
  host: 192.168.212.5
  user: admin  # Falls ein Benutzername benötigt wird
  password: Passwort  # Falls ein Passwort benötigt wird

database:
  path: /config/frigate.db

detectors:
  coral:
    type: edgetpu  
    device: usb

birdseye:
  enabled: True
  restream: True
  mode: objects

objects:
  track:
    - person

cameras:
  Hof:
    ffmpeg:
      inputs:
        - path: rtsp://admin:Passwort@192.168.212.105:554/cam/realmonitor?channel=1&subtype=0
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://admin:Passwort@192.168.212.105:554/cam/realmonitor?channel=1&subtype=1
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      width: 1280
      height: 720
      fps: 5
    objects:
      track:
        - person

    snapshots:
      enabled: true
      timestamp: true
      bounding_box: true
      retain:
        default: 30
    record:
      enabled: true
      events:
        retain:
          default: 14

  Tuer:
    ffmpeg:
      inputs:
        - path: rtsp://admin:Passwort@192.168.212.110:554/cam/realmonitor?channel=1&subtype=0
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://admin:Passwort@192.168.212.110:554/cam/realmonitor?channel=1&subtype=1
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      width: 1280
      height: 720
      fps: 5
    objects:
      track:
        - person

    snapshots:
      enabled: true
      timestamp: true
      bounding_box: true
      retain:
        default: 30
    record:
      enabled: true
      events:
        retain:
          default: 14

  Vorne:
    ffmpeg:
      inputs:
        - path: rtsp://admin:Passwort@192.168.212.107:554/cam/realmonitor?channel=1&subtype=0
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://admin:Passwort@192.168.212.107:554/cam/realmonitor?channel=1&subtype=1
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      width: 1280
      height: 720
      fps: 5
    objects:
      track:
        - person

    snapshots:
      enabled: false
      timestamp: true
      bounding_box: true
      retain:
        default: 30
    record:
      enabled: true
      events:
        retain:
          default: 14

  Hinten:
    ffmpeg:
      inputs:
        - path: rtsp://admin:Passwort@192.168.212.106:554/cam/realmonitor?channel=1&subtype=0
          input_args: preset-rtsp-restream
          roles:
            - record
        - path: rtsp://admin:Passwort@192.168.212.106:554/cam/realmonitor?channel=1&subtype=1
          input_args: preset-rtsp-restream
          roles:
            - detect
    detect:
      width: 1280
      height: 720
      fps: 5
    objects:
      track:
        - person

    snapshots:
      enabled: false
      timestamp: true
      bounding_box: true
      retain:
        default: 30
    record:
      enabled: false
      events:
        retain:
          default: 14

version: 0.14
`

**Bei den Kameras Hinten und Vorne 192.168.212.106 und 192.168.212.107 möchte ich PTZ und Autotracking aktivieren. Beide Kameras sind die Dahua DH-SD1A404XB-GNR und unterstützen [ONVIF](https://www.onvif.org/member-tools/wp-content/uploads/sites/2/2022/12/ONVIF_DoC_DH-SD1A404XB-GNR_2.812.0000032.2.R-Build-Date-2022-08-04_2022-11-25_15h8m25s.pdf)**

**_I want to activate PTZ and autotracking for the rear and front cameras 192.168.212.106 and 192.168.212.107. Both cameras are the Dahua DH-SD1A404XB-GNR and support [ONVIF](https://www.onvif.org/member-tools/wp-content/uploads/sites/2/2022/12/ONVIF_DoC_DH-SD1A404XB-GNR_2.812.0000032.2.R-Build-Date-2022-08-04_2022-11-25_15h8m25s.pdf)_**

**Diese Konfiguration habe ich in der [Frigate Dokumentation](https://docs.frigate.video/configuration/autotracking/) gefunden und habe diese mit den Angaben meiner Kameras ergänzt, leider funktioniert das auch nicht.**

**_I found this configuration in the [Frigate Doku](https://docs.frigate.video/configuration/autotracking/) and supplemented it with the information from my cameras, unfortunately that doesn't work either._**

`cameras:
  ptzcamera:
    ...
    onvif:
      # Required: host of the camera being connected to.
      host: 0.0.0.0
      # Optional: ONVIF port for device (default: shown below).
      port: 8000
      # Optional: username for login.
      # NOTE: Some devices require admin to access ONVIF.
      user: admin
      # Optional: password for login.
      password: admin
      # Optional: PTZ camera object autotracking. Keeps a moving object in
      # the center of the frame by automatically moving the PTZ camera.
      autotracking:
        # Optional: enable/disable object autotracking. (default: shown below)
        enabled: False
        # Optional: calibrate the camera on startup (default: shown below)
        # A calibration will move the PTZ in increments and measure the time it takes to move.
        # The results are used to help estimate the position of tracked objects after a camera move.
        # Frigate will update your config file automatically after a calibration with
        # a "movement_weights" entry for the camera. You should then set calibrate_on_startup to False.
        calibrate_on_startup: False
        # Optional: the mode to use for zooming in/out on objects during autotracking. (default: shown below)
        # Available options are: disabled, absolute, and relative
        #   disabled - don't zoom in/out on autotracked objects, use pan/tilt only
        #   absolute - use absolute zooming (supported by most PTZ capable cameras)
        #   relative - use relative zooming (not supported on all PTZs, but makes concurrent pan/tilt/zoom movements)
        zooming: disabled
        # Optional: A value to change the behavior of zooming on autotracked objects. (default: shown below)
        # A lower value will keep more of the scene in view around a tracked object.
        # A higher value will zoom in more on a tracked object, but Frigate may lose tracking more quickly.
        # The value should be between 0.1 and 0.75
        zoom_factor: 0.3
        # Optional: list of objects to track from labelmap.txt (default: shown below)
        track:
          - person
        # Required: Begin automatically tracking an object when it enters any of the listed zones.
        required_zones:
          - zone_name
        # Required: Name of ONVIF preset in camera's firmware to return to when tracking is over. (default: shown below)
        return_preset: home
        # Optional: Seconds to delay before returning to preset. (default: shown below)
        timeout: 10
        # Optional: Values generated automatically by a camera calibration. Do not modify these manually. (default: shown below)
        movement_weights: []`

**So und dies ja dann die bereinigte Version ohne Post Arguments / _So this is the cleaned up version without post arguments_**

`mqtt:
  host: 192.168.212.5

detectors:
  coral:
    type: edgetpu  
    device: usb

birdseye:
  enabled: True
  restream: True
  mode: objects

cameras:
  ptzcamera:
    ...
    onvif:
      host: 192.168.212.106   
      port: 80              
      user: admin             
      password: Passwort         
      autotracking:           
        enabled: False        
        calibrate_on_startup: False
        zooming: disabled
        zoom_factor: 0.3
        track:
          - person
        required_zones:
          - zone_name
        return_preset: home
        timeout: 10
        movement_weights: []`

**Das Ergebnis ist dann diese Fehlermeldung / _The result is then this error message_**

`Config Error:

Traceback (most recent call last):
  File &#34;/opt/frigate/frigate/api/app.py&#34;, line 245, in config_save
    FrigateConfig.parse_raw(new_config)
  File &#34;/opt/frigate/frigate/config.py&#34;, line 1664, in parse_raw
    config = load_config_with_no_duplicates(raw_config)
  File &#34;/opt/frigate/frigate/util/builtin.py&#34;, line 116, in load_config_with_no_duplicates
    return yaml.load(raw_config, PreserveDuplicatesLoader)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/__init__.py&#34;, line 81, in load
    return loader.get_single_data()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/constructor.py&#34;, line 49, in get_single_data
    node = self.get_single_node()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 36, in get_single_node
    document = self.compose_document()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 55, in compose_document
    node = self.compose_node(None, None)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 133, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/composer.py&#34;, line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/parser.py&#34;, line 98, in check_event
    self.current_event = self.state()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/parser.py&#34;, line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/scanner.py&#34;, line 116, in check_token
    self.fetch_more_tokens()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/scanner.py&#34;, line 223, in fetch_more_tokens
    return self.fetch_value()
  File &#34;/usr/local/lib/python3.9/dist-packages/yaml/scanner.py&#34;, line 577, in fetch_value
    raise ScannerError(None, None,
yaml.scanner.ScannerError: mapping values are not allowed here
  in &#34;&lt;unicode string&gt;&#34;, line 17, column 10:
        onvif:
             ^`

**Wie gesagt, dass ist die offizielle Vorgabe von Frigate bezüglich dem Autotracking / _As I said, this is Frigate's official requirement for auto tracking_**


**Ich habe in HomeAssistant versucht mal die Kamera via ONVIF anzusteuern bzw ob ich die darüber erreichen kann, das funktioniert. Nun habe ich gelesen, das man evtl. über HomeAssistant die AutoTracking und PTZ funktion konfigurieren muß-als Client. Dazu finde ich aber noch weniger im Internet. Das kann doch nicht so schwer sein die PTZ Steuerung der beiden Kameras zu realisieren und das die einem erkannten Objekt folgen und danach wieder auf die ursprüungliche Version zurück fahren.** 

**_I tried to control the camera via ONVIF in HomeAssistant or see if I could reach it via that, it worked. Now I have read that you may have to configure the AutoTracking and PTZ functions via HomeAssistant - as a client. But I can find even less about this on the internet. It can't be that difficult to implement the PTZ control of the two cameras and that they follow a recognized object and then return to the original version._**

**Comment by NexGen-3D-Printing** [comment]
I got this camera working properly today, below is a sample of my config, you need to make sure everything is correct or you will see errors and crashing, Frigate is really touchy, the formatting has to be perfect, Frigate will Frigout even if a low flying unlaiden Swallow burns past your house at less than optimal air speed, no matter if its European or African...

   Roundabout:
    ffmpeg:
      hwaccel_args: preset-vaapi
      inputs:
        - path: rtsp://127.0.0.1:8554/Roundabout_sub
          input_args: preset-rtsp-restream
          roles:
            - detect
        - path: rtsp://127.0.0.1:8554/Roundabout
          input_args: preset-rtsp-restream
          roles:
            - record
      output_args:
        record: preset-record-generic-audio-aac
    live:
      stream_name: Roundabout
    detect:
      width: 854
      height: 480
      fps: 10

    zones:
      Intersection:
        coordinates: 0.001,0.002,0.002,0.997,0.997,0.995,0.999,0.007
        loitering_time: 0
        objects: person
  
   onvif:
      host: 192.168.0.196
      port: 80
      user: user
      password: password
      tls_insecure: true
      autotracking:
        enabled: true
        calibrate_on_startup: false
        zooming: relative
        track:
          - person
        required_zones:
          - Intersection
        return_preset: Home
        timeout: 10

You need to make sure spaces and formatting are correct, along with creating a zone for the camera and using that name in this config, also you need to create a profile in your camera for a homing position, and use that profile name in the "return_preset:"

Also if something is optional, best to leave that out until you have it working.
